packages/hub/routes/checkly-webhook.ts

```diff
@@ -38,19 +38,19 @@ export default class ChecklyWebhookRoute {
     },
     'xdai archive health check (eth_blockNumber)': {
       componentName: 'Archive RPC node',
-      incidentName: 'RPC Node Degradation',
+      incidentName: 'Transactions delayed',
       incidentMessage:
-        'We are experiencing degraded service with our archive RPC node. This will result in reduced transaction processing times.',
+        'We are experiencing degraded service with our archive RPC node. This will result in increased transaction processing times.',
     },
     'xdai non-archive health check - late-cold-smoke (eth_blockNumber)': {
       componentName: 'RPC node (non-archive)',
-      incidentName: 'RPC Node Degradation',
+      incidentName: 'Transactions delayed',
       incidentMessage:
         'We are experiencing degraded service with our non-archive RPC node. This will result in reduced application performance.',
```

Comment:
Unsure about the implications of "reduced application performance" here, outside of transactions. I think the DApp still might try to request some data from the layer 2 chain directly, but for the most part we are getting our data from the subgraph.

Reply:
Yeah, but the subgraph gets its information from the rpc node that the warning is about.

Reply:
As well as the relay server uses the same rpc node the warning is about. Basically when this node goes down, so does the relay server, which means the apps will not be able to send any txns to layer 2